### PR TITLE
[Verification 1] Feature verification conductor app and database configuration on storage cluster

### PR DESCRIPTION
**Pull request by rwrzesien**
Resolves https://github.com/golemfactory/concent/issues/207.

Was a part of https://github.com/golemfactory/concent/pull/217

### CODE REVIEW

File: concent_api/conductor/models.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                'result_package_path': 'result_package_path cannot be used as source_package_path in any other VerificationRequest.'
+            })
+
+
+class BlenderSubtaskDefinition(Model):
+    """
+    For each VerificationRequest there must be exactly one BlenderSubtaskDefinition in the database.
+    """
+
+    class OutputFormat(ChoiceEnum):
```
Comment (dybi) [inline concent_api/conductor/models.py]: aren't there more formats allowed?[blender docs]( https://docs.blender.org/manual/en/dev/data_system/files/media/image_formats.html#image-formats)

Comment (dybi) [inline concent_api/conductor/models.py]: Are we supposed to support just those two formats? In the [blender docs]( https://docs.blender.org/manual/en/dev/data_system/files/media/image_formats.html#image-formats) seems to be more of it ;)

Comment (rwrzesien) [reply concent_api/conductor/models.py]: Off course we should, I was looking for this but couldn't find it, thanks for the link !

Comment (cameel) [reply concent_api/conductor/models.py]: Let's drop EXR.

Also, please make sure that both uppercase and lowercase versions are supported.

Comment (rwrzesien) [reply concent_api/conductor/models.py]: done :+1: 

What do you mean by `are supported` ?

File: concent_api/conductor/models.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    """
+
+    # Relative path of the file. Relative to the same directory that paths listed in FileTransferTokens are relative to.
+    path = CharField(max_length=MESSAGE_PATH_LENGTH)
+
+    # Foreign key to VerificationRequest. Can be NULL if there's no corresponding request.
+    verification_request = ForeignKey(VerificationRequest, related_name = 'upload_reports', blank=True, null=True)
+
+    # Indicates when conductor has been notified about the upload.
+    created_at      = DateTimeField(default = timezone.now)
```
Comment (Jakub89) [inline concent_api/conductor/models.py]: There seams to be a lot of "space" issue i this whole PR. Which style we should use anyway? With or without spaces in function call?

Comment (rwrzesien) [reply concent_api/conductor/models.py]: @Jakub89 All should be updated by now.

File: concent_api/conductor/models.py (hunk 1 above)
Comment (cameel) [reply concent_api/conductor/models.py]: I mean that `TaskToCompute` can specify the format as either `PNG` or `png` and both are accepted by Golem (see https://github.com/golemfactory/golem/issues/218). And maybe things like `PnG` are accepted too. We should be able to handle all values accepted by Golem.

Though not necessarily in this enum - we could assume that Concent e.g. converts extension to lower case when it passes the format in a Celery task.

Comment (rwrzesien) [reply concent_api/conductor/models.py]: :+1: Off course, but that will be handled in the following pull requests, here is only definition.

**Pull request merged**